src/main/java/org/basseur/adventofcode/advent2018/utils/FileReaders.java

```diff
@@ -17,14 +18,18 @@
 @Component
 public class FileReaders {
 
+    @Autowired
+    ResourceLoader resourceloader;
+
     /**
      * Parses the input file line by line into an array of {@code Integer}s and returns it.
      *
      * @param filename path to the input file
      * @return an array of integers
      */
     public Integer[] readFileIntoArrayOfIntegers(String filename) {
-        Scanner scanner = openFileReturnScanner(filename);
+        InputStream fileInputStream = FileReaders.class.getResourceAsStream(filename);
```

Comment:
We need to load as an inputStream and process directly, loading from a path does not work here: https://stackoverflow.com/questions/39613848/resource-is-not-loading-in-executable-spring-boot-jar-how-executable-jar-loads (see first post and the "here is the solution" part)